nef-plugin/SourceEditorExtension.swift

```diff
@@ -28,15 +28,15 @@ class SourceEditorExtension: NSObject, XCSourceEditorExtension {
         [.preferences,
          .exportSnippet(selection: ""),
          .exportSnippetToClipboard(selection: ""),
-         .markdownPage(playground: ""),
+         .markdownPage(page: ""),
          .playgroundBook(package: "")]
```

Comment:
I mentioned it in another review, but here it becomes even more obvious. This is an incorrect model if you need to create values with empty data to represent something.

Reply:
it will be propagated after merging the previous PR